data/competitors.yaml

```
@@ -104,19 +104,19 @@ competitors:
     is_synthorg: true
     features:
       org_structure: { support: full, note: "Departments, hierarchy, 8 seniority levels, reporting lines, HR" }
-      multi_agent: { support: planned, note: "6 assignment strategies, conflict resolution; working toward multi-project support and formal coordination models" }
+      multi_agent: { support: planned, note: "6 assignment strategies, 4 coordination dispatchers, conflict resolution shipped; multi-project support and formal coordination models in progress" }
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)